qt-models/diveplannermodel.cpp

```diff
@@ -1154,8 +1155,13 @@ void DivePlannerPointsModel::createPlan(bool replanCopy)
 		Command::addDive(&displayed_dive, false, false);
 	} else {
 		// we were planning an old dive and rewrite the plan
-		mark_divelist_changed(true);
+		displayed_dive.maxdepth.mm = 0;
+		displayed_dive.dc.maxdepth.mm = 0;
+		fixup_dive(&displayed_dive);
 		copy_dive(&displayed_dive, current_dive);
+		mark_divelist_changed(true);
+		emit updateDiveInfo();
```

Comment:
That signal seems not to be caught anywhere...?

Reply:
@sfuch79: could you please shed light on this signal thing? I don't see where this signal is caught - am I missing something?

Moreover, I think this could use a CHANGELOG.md entry as it fixes a bug.